src/Strategy/Node/Errors.hs

```diff
@@ -0,0 +1,42 @@
+module Strategy.Node.Errors (
+  MissingNodeLockFile (..),
+  fossaNodeDocUrl,
+  npmLockFileDocUrl,
+  yarnLockfileDocUrl,
+) where
+
+import App.Docs (strategyLangDocUrl)
+import Data.Text (Text)
+import Diag.Diagnostic (ToDiagnostic, renderDiagnostic)
+import Prettyprinter (Pretty (pretty), indent, vsep)
+
+yarnLockfileDocUrl :: Text
+yarnLockfileDocUrl = "https://classic.yarnpkg.com/lang/en/docs/yarn-lock/"
```

Comment:
We should include references to yarn v2 as well.  I don't think we'll have a way of knowing which one was used, so we should default to always showing both.

Reply:
fixed in: https://github.com/fossas/fossa-cli/pull/805/commits/c64f7b9247ca7f0d676674640ce83ccdda5c2e34